code_folder/C0_data_processing.py

```diff
@@ -584,6 +730,14 @@ def evaluate_lifetime_costs(settings, economic_data, dict_asset):
             dict_asset["opex_var"]["value"], int
         ):
             opex_var = dict_asset["opex_var"]["value"]
+        # if multiple busses are provided, it takes the first opex_var (corresponding to the first bus)
```

Comment:
Ooh, you ment price changes in opex_var. Will comment this out later.

Reply:
We have to discuss this again.
Did I code this with lifetime_opex_var and lifetime_capex_var? I did not find it in the master branch, so I am not sure. 

I actually believe that capex_var can not be a timeseries, and I am not sure as to why to calculate the average. Same for opex_var. lifetime_opex_var also does not seem to be used below?

the only way a calculatin of lifetime_opex_var would make sense if we included energy price changes, which we currently do not do.